### ISSUE TITLE
Broken GitHub actions 

### ISSUE THREAD

**Issue by rodericj** [AppPear/ChartView#124]
## Description
There are some failures in the GitHub actions. 

## Expected Behavior
Actions should pass 

## Actual Behavior
Actions do not pass

## Possible Fix
The issues seem to have solution suggestions. Things like platform checks should be pretty straightforward. 

## Steps to Reproduce
Create a Pr

## Your Environment
GitHub 

* Version of this package used: latest
* Device/Simulator: CI
* Operating System and version:
* Link to your project:

**Comment by AppPear** [comment]
I know, if you can come up with a fix it would be welcomed! I couldn't figure it out how I could build this swift package without pushing a xcode project file

**Comment by rodericj** [comment]
Ill see if I can take a shot at it this week

**Comment by LucasCarioca** [comment]
Seems like one of the issues is with SF Symbols. 
Since it is being built on macOS and not for iOS those are not available. 

I did find a tool that might help: https://github.com/davedelong/sfsymbols

Not sure if that would work but might be worth a shot

**Comment by satan87** [comment]
SF Symbols will be available on mac at the next release.
Maybe it's better to just de-activate the macOS target till the next release?

**Comment by LucasCarioca** [comment]
Yeah that's a good idea. I think a lot of the issues on there are due to the macOS target.

**Comment by rodericj** [comment]
Indeed @LucasCarioca I think you are correct. I've started a PR to explore this: https://github.com/AppPear/ChartView/pull/127.

**Comment by LucasCarioca** [comment]
Looking at that last run, we're using a mix of macOS stuff in the code. 

We will need to replace all of that to get it to build. To make this cross platform we will have to isolate all of that and include logic to determine when it's used.

**Comment by rodericj** [comment]
My diff seems to be not failing if anyone wants to take a look.

**Comment by LucasCarioca** [comment]
looks promising!
